app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I added it because I was copying from tasks_controller, and thought it was some sort of convention ;) Happy to remove it.